File: Lab1_Ex/Lab1B.java
/*
Class: CSE 1321L
Section: ...
Term: Summer 2022
Instructor: ...
Name: Nasimul Labib
Lab#: 1B
*/ 


// Program Lab1B.java
// Demonstrate reading a string from the user.
import java.util.Scanner;
public class Lab1B
{
 // Reads a character string from the user and prints it.
 public static void main (String[] args)
 {
 String message;
 Scanner scan = new Scanner (System.in);
 System.out.print ("Enter a line of text: ");
 message = scan.nextLine();
 System.out.println ("You wrote \' " + message + "\'");
 }
} 

File: Lab1_Ex/Lab1B.cs
/*
Class: CSE 1321L
Section: w2
Term: Summer 2022
Instructor: Kavitha Muppa
Name: Nasimul Labib
Lab#: 1A
*/ 
using System;
// Program Lab1B.cs
// Demonstrate reading a string from the user.
public class Lab1B
{
 // Reads a character string from the user and prints it.
 public static void Main (String[] args)
 {
 String message;
 Console.Write("Enter a line of text: ");
 message = Console.ReadLine();
 Console.WriteLine("You wrote'" + message + "'");
 }
} 
